tokio-postgres/src/simple_query.rs

```diff
@@ -98,7 +98,12 @@ impl Stream for SimpleQueryStream {
                         .parse()
                         .unwrap_or(0);
                     let fields = if *this.include_fields_in_complete {
-                        this.fields.clone()
+                        let _tag = body.tag().expect("Failed to get tag");
```

Comment:
The `_var` syntax is only used when `_var` is unused. This should just be `tag`. 

You could also do `if body.tag().expect("Failed to get tag").starts_with("SELECT") { }`. Long chains of function calls are acceptable in rust.